common/smoke-tests/README.md

```diff
@@ -75,7 +75,7 @@ All the classes in this sample not depend on each other.
 It is possible to run them individually:
 ```java
 package com.azure;
-import java.io.IOException;
+import com.azure.smoketest.StorageBlob;import java.io.IOException;
```

Comment:
Typo?

Reply:
Good catch.
Corrected the package to com.azure to com.azure.smoketest and removed "import com.azure.smoketest.StorageBlob".